maths/gaussian.py

```diff
@@ -21,7 +21,7 @@ def gaussian(x, mu: float = 0.0, sigma: float = 1.0) -> int:
     
     >>> gaussian(1,4,3)
     0.0806569081730478
-
+    
```

Comment:
whitespace error.

Reply:
That is an older version. I have updated and removed all the white spaces.